README.dev.md

```diff
@@ -111,54 +111,61 @@ Otherwise, please ensure check both linting (`ruff fix .`) and formatting (`ruff
 
 We use [prettier](https://prettier.io/) for formatting most other files. If you are editing or adding non-python files and using VS code, the [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) can be installed to auto-format these files as well.
 
-# The following sections are untested
-
-## Generating the API docs
-
-```shell
-cd docs
-make html
-```
-
-The documentation will be in `docs/_build/html`
-
-If you do not have `make` use
-
-```shell
-sphinx-build -b html docs docs/_build/html
-```
-
-To find undocumented Python objects run
-
-```shell
-cd docs
-make coverage
-cat _build/coverage/python.txt
-```
-
-To [test snippets](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html) in documentation run
-
-```shell
-cd docs
-make doctest
-```
-
 ## Making a release
 
+### Automated release workflow:
+
+0. **IMP0RTANT:** Create a PR for the release branch and make sure that all checks pass!
+   - if everything goes well, this PR will automatically be closed.
+1. Navigate to [Draft Github Release](https://github.com/EIT-ALIVE/eitprocessing/actions/workflows/release_github.yml)
+   on the [Actions](https://github.com/EIT-ALIVE/eitprocessing/actions) tab.
+2. On the right hand side, you can select the level increase (patch, minor, or major) and which branch to release from.
+   - if released from a different branch than `develop`, then the workflow will attempt to merge the changes into develop
+     as well. If succesfull, the release branch will be deleted from the remote repository.
+   - [follow semantic versioning conventions](https://semver.org/) to chose the level increase:
+     - `patch`: when backward compatible bug fixes were made
+     - `minor`: when functionality was added in a backward compatible manner
+     - `major`: when API-incompatible changes have been made
```

Comment:
I cannot run the workflow from the Actions tab (it doesn't show the workflow_dispatch). Is this because it is not in the main?

Reply:
indeed, I commented on this in the top comment of this PR with an explanation of how to run it from commandline